routes/mail/index.js

```diff
@@ -1,32 +1,28 @@
-"use strict";
+import request from 'superagent';
+const slackPostAPI = process.env.KINDLIZED_SLACK;
```

Comment:
__[jshint]__
*  'const' is available in ES6 (use esnext option) or Mozilla JS extensions (use moz).